editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -1574,6 +1574,8 @@ VersionControlEditorPlugin::VersionControlEditorPlugin() {
 
 VersionControlEditorPlugin::~VersionControlEditorPlugin() {
 	shut_down();
+	memdelete(metadata_dialog);
+	memdelete(set_up_dialog);
```

Comment:
Seems fine for now, but this should likely be refactored at some point to be like other editor plugins. Normally `*EditorPlugin` classes have a dedicated root node which owns everything, and whose lifecycle is handled by EditorNode (I think).